php/util.php

```diff
@@ -5,8 +5,13 @@
 require_once( $rootPath.'/conf/config.php' );
 
 // Automatically include only the used utility classes
-spl_autoload_register(function ($class) {
-    require_once 'utility/'. strtolower($class). '.php';
+spl_autoload_register(function ($class) 
+{
+	// Remove namespaces from the classname string
+	// Important for compatibility with 3rd party plugins
+	$class = end(explode('\\',$class));
```

Comment:
PHP Notice:  Only variables should be passed by reference
I.e. we should use temporary variable for 'explode' result.